rnndb/nvchipsets.xml

```diff
@@ -92,6 +92,7 @@
 
 	<value value="0x117" name="GM107"/>
 	<value value="0x118" name="GM108"/>
+	<value value="0x120" name="GM200"/>
```

Comment:
Please put this after GM204. Or update all of the places that currently refer to GM204 to instead refer to GM200.